card-flow.md

```diff
@@ -0,0 +1,52 @@
+# Project Management Boards
+
+Our team uses project boards to manage our projects and track our progress.  The boards serve two purposes: to allow clients to see what's happening and give feedback on a project, and to allow us to manage our team, our tasks, and stay coordinated. Each project has its own board. The board shows us the current state of all the things we're working on in our project and where each task is in the lifecycle of development. Each task is represented by a card on the board, and as cards move through the development process, we move them to different columns on the board.
```

Comment:
I think it might be good to clarify this a bit.  So look for a way to mention Kanban (see below).  It's totally fine to mention concrete examples of the tools such as Trello or AgileZen.  Once FC is ready, we'll rewrite this to accurately reflect card flow within FC.  

Our team uses a web-based Kanban planning tool, such as Trello or AgileZen, to manage our projects and track our progress.